charmander/tests/stores/test_example_store.py

```diff
@@ -0,0 +1,89 @@
+"""
+Example persistence tests.
+
+Tests cover model-specific constraints under the assumption that framework conventions
+handle most boilerplate.
+
+"""
+from hamcrest import (
+    assert_that,
+    calling,
+    equal_to,
+    is_,
+    raises,
+)
+from microcosm_postgres.context import SessionContext, transaction
+from microcosm_postgres.errors import DuplicateModelError
+
+from charmander.app import create_app
+from charmander.models.example_model import Example
+
+
+class TestExamples:
```

Comment:
Generally if we do want to test any complicated persistence-related behavior we do that in the tests for the store.